data/operators/amenity/drinking_water.json

```diff
@@ -0,0 +1,22 @@
+{
+  "properties": {
+    "path": "operators/amenity/drinking_water",
+    "skipCollection": true,
```

Comment:
I am guessing this line refers to the planet collection? If so, I expect it should be removed?

Reply:
Yes I think so.. If you remove this line and run the build again, it will bring in all the "amenity/drinking_water" items found in the planet.  There are a lot of them and they look mostly useful 👍 

https://github.com/osmlab/name-suggestion-index/blob/main/dist/collected/operators_all.json#L3643-L3737